markdown_stoplight/changelog.md

```diff
@@ -2,164 +2,236 @@
 
 This change log covers significant updates and changes to the Beyond REST API.
 
+**2019-05-09**
+
+**Update**
+
+* Added `productIdentifiers` property to [Show variation details](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/products/variation-get) and [Update variation partially](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/products/variation-patch-json).
+
+* Added `workflow` property to [Show payment method details](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/payment-methods/payment-method-get).
+
+* Added query parameter `fileName` and header `Content-Type` to request parameters of [Add product image](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/products/product-images-create).
+
+* Added `bankName`, `iban`, `bic`, `bankAccount`, and `bankIdentifier` properties to [Show legal details](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/shop/merchant-legal-get) and [Update legal resource partially](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/shop/merchant-legal-jsonpatch) resources.
+
+**2019-04-23**
+
+**New**
+
+Add *Events* section. This section explains how applications can receive information about customer actions in ePages online shops.
```

Comment:
Is the changelog refering to http://docs.beyondshop.cloud or https://beyond.docs.stoplight.io/ ?

Even though all the other links are referring to Stoplight, I can't find the "Events" section there.

Reply:
It may be good to link to this as well.

Reply:
The examples in the events section refer to ePages Now:

```
checkoutUrl: "https://pm.epages.com/epages/apidocu.sf/?ObjectPath=/Shops/apidocu/AnonymousUsers/1/Baskets/46072&ChangeAction=PickupBasket&PickupToken=MjNiZThlOGM0MjAzZWQ0N2ZjYmZmZDFiZjI3OTQxMzkwOWY4ZjZlOGE4NTM0ZWIxMjg3NmY2NzBiY2IxYzQ1OV8xNDgyMjIzMDcz"
```

Reply:
Are you sure that we even provide this functionality for Beyond? It's the first time that I take notice of it.

Reply:
Yes, we provide this for Beyond. But you're right, the examples require an update for Beyond.